mod-invoice-storage/examples/invoice.sample

```diff
@@ -2,17 +2,22 @@
   "id": "733cafd3-895f-4e33-87b7-bf40dc3c8069",
   "adjustments": [
     {
-      "adjustmentLabel": "Sales Tax",
-      "adjustmentType": "Percentage",
-      "adjustmentValue": 6
+       "id":"14263aab-b22b-4ddc-9ecc-3434427c2c8f"
```

Comment:
invoice adjustments which aren't prorated and are "in addition to" the total... this is where you'd want (need?) a fundDistribution.  We need to know which fund to pay this adjustment from.